pkl-core/src/main/java/org/pkl/core/evaluatorSettings/PklEvaluatorSettings.java

```diff
@@ -38,6 +38,7 @@ public record PklEvaluatorSettings(
     @Nullable Map<String, String> env,
     @Nullable List<Pattern> allowedModules,
     @Nullable List<Pattern> allowedResources,
+    @Nullable Boolean color,
```

Comment:
Blocking: feels like this should be `@Nullable Color color` (where we move enum `Color` from `pkl-commons-cli`)

I think we should turn that enum into a boolean color right before passing this option to `EvaluatorBuilder`.